app/services/the_tvdb_service/row.rb

```diff
@@ -0,0 +1,87 @@
+class TheTvdbService
+  class Row
+    attr_reader :media_id, :media_type, :data, :episode_id, :tvdb_series_id
+
+    def initialize(media_id, data, tvdb_series_id)
+      @media_id = media_id
+      @data = data
+      @episode_id = data['id']
+      @tvdb_series_id = tvdb_series_id
+      @media_type = 'Anime'
+    end
+
+    def update_episode
+      return unless episode_number.present?
+
+      %i[
```

Comment:
You should put a comment here explaining that all of these methods will modify the `episode` object in-place